testbenches/simlibs.mk

```diff
@@ -177,9 +183,9 @@ VLOGAN_VFLAGS   += -ntb_opts tb_timescale=1ps/1ps -timescale=1ps/1ps \
 # with `make squeakyclean` or an equivalent
 $(TESTBENCH_PATH)/synopsys_sim.setup: $(TESTBENCH_PATH)/gen_simlibs.tcl
 	cd $(TESTBENCH_PATH) && vivado -mode batch -source $<
-
-$(WORKDIR)/AN.DB: $(TESTBENCH_PATH)/synopsys_sim.setup $(CL_DIR)/Makefile.machine.include $(VHEADERS) $(VSOURCES)
 	echo "$(PROJECT) : $(WORKDIR)/64" >> $(TESTBENCH_PATH)/synopsys_sim.setup
+
+$(WORKDIR)/AN.DB: $(TESTBENCH_PATH)/synopsys_sim.setup $(BSG_MACHINE_PATH)/Makefile.machine.include $(HARDWARE_MACHINE_FILE) $(VHEADERS) $(VSOURCES)
```

Comment:
HARDWARE_MACHINE_FILE is defined by Makefile.machine.include and is the file-path of machine.mk for a target machine